packages/terra-alert/src/terra-dev-site/test/alert/AlertPopup.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Can we rename this to remove popup? Sounds like the alert should be in a popup not rendered after an action

Reply:
Removed 8b24860